mcs/class/System.Net.Http/IMonoHttpClientHandler.cs

```diff
@@ -78,5 +78,8 @@ interface IMonoHttpClientHandler : IDisposable
 		}
 
 		Task<HttpResponseMessage> SendAsync (HttpRequestMessage request, CancellationToken cancellationToken);
+
+		// Only used by MonoWebRequestHandler and ignored by the other handlers.
+		void MonoSetTimeout (TimeSpan timeout);
```

Comment:
The comment above it explains it.   We can use something like "Legacy" instead, but I strongly suggest keeping the prefix because it is a "Mono-only" method that will have no effect in the other handlers.

Reply:
Use different prefix, it's not legacy either from the description it looks like it should be `SetWebRequestTimeout` or something like that